sequencer/src/main/java/net/consensys/linea/rpc/LineaEstimateGas.java

```diff
@@ -189,18 +189,7 @@ private Wei getEstimatedPriorityFee(
         txProfitabilityCalculator.profitablePriorityFeePerGas(
             transaction, profitabilityConf.estimateGasMinMargin(), estimatedGasUsed, minGasPrice);
 
-    if (profitablePriorityFee.greaterOrEqualThan(priorityFeeLowerBound)) {
```

Comment:
I thought it's a part of the previous PR

Reply:
solved